docs/book/src/SUMMARY.md

```diff
@@ -19,16 +19,25 @@
   - [What's in a controller?](./cronjob-tutorial/controller-overview.md)
   - [Implementing a controller](./cronjob-tutorial/controller-implementation.md)
 
-      - [You said something about main?](./cronjob-tutorial/main-revisited.md)
+    - [You said something about main?](./cronjob-tutorial/main-revisited.md)
 
+  - [Implementing webhook](./cronjob-tutorial/webhook-implementation.md)
```

Comment:
Call this something like `Implementing validating/defaulting webhooks` or something slightly more descriptive (also grammar nit: webhooks should be plural (note the `s`))